td/imports/management/commands/language_alt_names.py

```diff
@@ -0,0 +1,86 @@
+"""
+
+USAGE:
+    python manage.py language_alt_names <username> < <file>.csv
```

Comment:
It might be good to use square brackets around `username` and `file`.  It took a couple minutes before I realized the csv file was redirected input rather than another parameter.